### PR TITLE
Implement profile page

### PR DESCRIPTION
**Pull request by yana617**

### CODE REVIEW

**Comment by yana617** [comment]
Blocked by PR16

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "axios": "^0.21.1",
     "core-js": "^3.6.5",
     "inputmask": "^5.0.5",
+    "moment": "^2.29.1",
```
Comment (Xoxol) [inline package.json:19]: You don't probably need moment.js. Saying more, the developers of moment suggest not to use moment: https://momentjs.com/docs/#/-project-status/

[Day.js](https://github.com/iamkun/dayjs) is pretty the same solution. But I would prefer [date-fns](https://date-fns.org/)

Comment (yana617) [reply package.json:19]: Antd Design DatePicker uses moment for props, I didn't want to use it. I will try to replace with date-fns, but I'm not sure will it work or not, not all works with Vue version 3.

Comment (yana617) [reply package.json:20]: @Xoxol 
Screen Shot 2021-06-21 at 02 38 08

**Review by Xoxol — APPROVED** [review]

**Pull request merged**